blivet/devices/lvm.py

```diff
@@ -1156,11 +1175,17 @@ def __init__(self, name, parents=None, size=None, uuid=None,
                                                 percent=percent,
                                                 segType=segType)
 
-        self.metaDataSize = metadatasize or 0
         self.chunkSize = chunksize or 0
         self.profile = profile
         self._lvs = []
 
+        if metadatasize:
+            self.metaDataSize = metadatasize or 0
```

Comment:
Actually, it should be moved before the ``if`` block to make sure ``self.metaDataSize`` is always set to something when ``__init__()`` finishes. Fixing.